### ISSUE TITLE
[Bug]: NSFW content unplayable when Family Friendly toggle turned off

### ISSUE THREAD

**Issue by hix1000** [FreeTubeApp/FreeTube#5579]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

NSFW videos don't play when Family Friendly content is turned off

### Expected Behavior

NSFW videos play as usual (possibly with a warning ahead of time), if Family Friendly content is turned off

### Issue Labels

accessibility issue, API issue, content not loading, feature stopped working, usability issue

### FreeTube Version

v0.21.3 Beta

### Operating System Version

Windows 11

### Installation Method

.exe

### Primary API used

Invidious API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
Okay so just to be clear you turned on the `Show Family Friendly Only` parental control setting and are surprised that FreeTube is not showing you NSFW videos?

**Comment by hix1000** [comment]
yes, and if I turn it off, Freetube refuses to play them.

**Comment by hix1000** [comment]
by Warn ahead of time, I mean we should have a disclaimer with the options to either skip or continue.

**Comment by absidue** [comment]
Those error messages sound like they are confusing from the Invidious API, but you said you were using the local API.

Without further information e.g. links to the videos or screenshots of the error messages I can't say exactly what is going on but the second error message likely means that the uploader disabled embedding, so the age-restriction cannot be bypassed and the only way to watch it is with an official YouTube app while logged in with an age-verified YouTube account.

**Comment by hix1000** [comment]
Now it says to watch on the latest version (also, my bad it's invidious)
![Screenshot 2024-08-21 160414](https://github.com/user-attachments/assets/20bed52f-09f0-477e-aadf-f7ae8bd0184f)

**Comment by hix1000** [comment]
Invidious Error: This video may be inappropriate for some users.
![Screenshot 2024-08-21 160818](https://github.com/user-attachments/assets/53f483b6-e888-4769-9936-adde8eb297e6)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> Now it says to watch on the latest version of Youtube? (also, my bad it's invidious) ![Screenshot 2024-08-21 160414](https://private-user-images.githubusercontent.com/114823290/360158304-20bed52f-09f0-477e-aadf-f7ae8bd0184f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1NjI4MTksIm5iZiI6MTcyNTU2MjUxOSwicGF0aCI6Ii8xMTQ4MjMyOTAvMzYwMTU4MzA0LTIwYmVkNTJmLTA5ZjAtNDc3ZS1hYWRmLWY3YWU4YmQwMTg0Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQxODU1MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mZWQ3MTkxZWMyMDUzOWJmODA4NTM5MjFmNjZhZmI4MjdjY2VlMGQ2MWZlNDMxOWJkY2E3NjkwYTExOTc2OWQwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.F3Nf_tCq973ErhwCI0HyyA8K6m0WFdaJVYBR3UIl2rc)

This one is unrelated, see https://github.com/FreeTubeApp/FreeTube/issues/5489



> Invidious API Error: This video may be inappropriate for some users. ![Screenshot 2024-08-21 160818](https://private-user-images.githubusercontent.com/114823290/360158656-53f483b6-e888-4769-9936-adde8eb297e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1NjI4MTksIm5iZiI6MTcyNTU2MjUxOSwicGF0aCI6Ii8xMTQ4MjMyOTAvMzYwMTU4NjU2LTUzZjQ4M2I2LWU4ODgtNDc2OS05OTM2LWFkZGU4ZWIyOTdlNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQxODU1MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xM2FlY2EwMTZiYWJjMzFjYTE1ZWZlM2YxMWVkZmUwMWUxNDU2ZWM4MTNjZWY3M2Y1MWIwZTI4OGE1ODJlZDFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.R4-91LK0duLY5u-0DPTCJWlpWl17UgzvKJEKwnF2MLQ)

See https://github.com/iv-org/invidious/issues/2189

Nothing we can do about this and these errors are unrelated to that setting.
